sdks/python/apache_beam/runners/common.py

```diff
@@ -197,14 +197,19 @@ def create_invoker(
           signature, context, side_inputs, input_args, input_kwargs)
 
   def invoke_process(self, windowed_value, restriction_tracker=None,
-                     output_processor=None):
+                     output_processor=None,
+                     additional_args=None, additional_kwargs=None):
     """Invokes the DoFn.process() function.
 
     Args:
       windowed_value: a WindowedValue object that gives the element for which
                       process() method should be invoked along with the window
                       the element belongs to.
       output_procesor: if provided given OutputProcessor will be used.
+      additional_args: additional arguments to be passed to the process()
+                       invocation, usually as side inputs.
+      additional_kwargs: additional keyword arguments to be passed to the
+                         process method.
```

Comment:
<!--thread_id:cc_170393592_t; commit:832655fe9210190bb88183127c4aece1a5e7549e; resolved:1-->
<!--section:context-quote-->
> **chamikaramj** wrote:
> Updated documentation a bit. Args for constructor is for all process() invocations while args here is for the current process invocation. (this was required since in SDF we pass arguments to the ProcessFn (a DoFn) that are already resolved directly to the corresponding SDF invocation.)

<!--section:body-->
Done.